darkvector.less

```diff
@@ -1210,3 +1214,38 @@ body.ltr #footer #footer-places {
     overflow: hidden;
     height: 0px
 }
+
+.tocnumber {
+list-style-type:inherit;
+background-color: transparent;
+border-color:#666;
+color:#af9f84
+}
+.toctogglelabel {
+    cursor: pointer;
+    color: #af9f84
+}
+.mw-changeslist-legend {
+     color: #a9a9a9;
+     background-color: #404040
+}
+ol.references li:target { 
+    background-color: #663300 
+}
+sup.reference:target { 
+    background-color: #663300 
+}
+.mw-editfont-monospace {
+    background-color: #393939
+}
+.diff.diff-contentalign-left {
+    background-color: #393939
+}
+.catlinks {
+    background-color: #393939
+}
+.mw-createacct-benefits-list p{
+    color: #c9c9c9
+}
+div#mw-panel div.portal div.body ul li a:visited{color:#af9f84}
+div#mw-panel div.portal div.body ul li a{color:#e69710}
```

Comment:
Actually, the `a` tag itself has a specific color. Is it really so important to duplicate these styles?

Reply:
Users should get the same functionality they are used to.  In the Vector skin, mw-panel links behave just like the rest of the links on the page.

This section baffled me for a while because it seems there are several conflicting css instructions.  I think that code is from dark-css but I changed the visited color.

Reply:
I mean that instead of copying css rules to override style of some specific kind of hyperlinks (like the sidebar items), it's better to just remove css rules which prevents default hyperlink styles from being used.

Reply:
I had the toughest time trying to do it the way you are saying and everything I tried seemed to prevent the wiki page from loading.  Finally I gave up and used that code which I agree is ugly.  But at any rate, would it be easier if I broke this whole thing up into smaller pieces?  I don't want to get hung up on the changes we might disagree on when there are plenty that should be uncontroversial.

Reply:
If it is so hard to use just one style for all the hyperlinks on any page, that's OK, you can leave these lines as is. But please change the shade of visited hyperlink to darker one, just like I've mentioned in the comment above.